### DISCUSSION TITLE
scrum prep Milestone 1 - 3

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#73] [General]
Write your scrum here.

**Comment by Brandell13** [comment]
I finished my two documentation tasks and my LTT. We meet on Tuesday and Wednesday

**Comment by AlexisTraversoPR** [comment]
This week I worked on the documentation and finished a task about having a way to trigger notifications

**Comment by Janiel777** [comment]
Voy hacer una reunión hoy con el team de notificaciones para poder discutir y planear cuál va a ser nuestro plan para la semana que viene. Quiero primero escuchar de lo que se va hablar hoy en el scrum en la clase para poder planear mejor el tema de la reunión que voy hacer hoy. Además terminé mis tasks de documentación el 18 de septiembre.

**Comment by AleDiazz** [comment]
Completed two key documentation sections 3.2 Verify and 2.3.1 Implement, participated in group meetings

**Comment by Yahid1** [comment]
This week I ensured and reviewed all code Task, documentation and LTT. Established clear grading confusions and clarified all task questions. Helped give ideas on the documentation and organized some of the parts to make them presentable.

**Comment by AngelMartinez33** [comment]
- Write documentation tasks (domain rough sketch, domain description) and LTT
- Had blue team meeting and ensured every developer has enough tasks to complete
- Design infographics about grading calculations

**Comment by IvanierB** [comment]
This week I completed a Documentation task I started last week and a coding task to implement a button for the notifications tab.

**Comment by marcosarraiza** [comment]
This week I completed my documentation and lecture topic tasks, we also met on Tuesday and Wednesday.

**Comment by Alej24** [comment]
This week I worked on documentation tasks and attended two meetings.

**Comment by sebasagir** [comment]
This week I worked on the issue of improving the design of the notifications menu and on the documentation issue of domain terminology in relation to domain rough sketch

**Comment by JaseyAcevedo** [comment]
Finished documentation task making 2 lecture topic tasks for Milestone 2

**Comment by diegorobles2** [comment]
This week i finished my LTT and documentation tasks

**Comment by MrL9000** [comment]
This week I basically finished everything I had planned out: I identified any first order ignorance relevant to the Metrics team, I filled out the 1.2.1 section of the documentation, Current Situation, and I designed the main design template that  was later used by another team member to create the base design of the app. All things considered, this was a rocky but decent start, hopefully by Milestone 2 we're able to finally code the application.

**Comment by yamayrie** [comment]
I completed two documentation tasks for the section 2.2.1 user stories, epics, features and 2.2.2 personas. All information is found on issue #33 and #34. I also attended a general group meeting.

**Comment by juan-ortiz92** [comment]
This week I worked on documentation for the first milestone and worked on my first lecture topic task regarding domain isolation.

**Comment by JConcepcion01** [comment]
This week I worked on finishing my remaining two tasks

**Comment by rafa-quiles** [comment]
This week I attended meetings and finished my documentation task for milestone 1

**Comment by dylanDeJesus21** [comment]
Created an additional issue. Created an Angular playlist to close the issue. Worked with the lecture topic task.

**Comment by JoelTorres20** [comment]
This week I finished all of my opened tasks for the first milestone.

**Comment by warm007** [comment]
During this week, I attended the general meeting and team meetings. I also worked on three tasks: Menu Design (#26), where I created an intuitive and functional menu layout; Menu Interactive Buttons Design (#70), where I implemented interactive buttons to improve navigation and allow filtering by ownership; and Menu Roadmap (#74), where I developed a plan for implementing and enhancing menu features.
